scripts/trigger-pipeline.sh

```diff
@@ -38,13 +39,47 @@ cat > /tmp/swissknife/trigger_pipeline.sh <<'EOF'
   echo "Finished triggering pipeline"
   echo "----------------------------------------"
 EOF
+else
+cat > /tmp/swissknife/trigger_pipeline.sh <<'EOF'
+  #!/bin/bash -x
+  echo "----------------------------------------"
+  echo "Triggering Pipeline"
+
+  export vcs_type="$1";
+  export username="$2";
+  export reponame="$3";
+  export tag="$4";
+  export params="$5";
+
+  trigger_workflow() {
+    curl --silent -X POST \
+      "https://circleci.com/api/v2/project/$vcs_type/$username/$reponame/pipeline?circle-token=${CIRCLE_TOKEN}" \
+      -H 'Accept: */*' \
+      -H 'Content-Type: application/json' \
+      -d '{
+        "tag": "'$tag'",
+        "parameters": '"$params"'
+      }'
+  }
+
+  trigger_workflow
+
+  echo "Finished triggering pipeline"
+  echo "----------------------------------------"
+EOF
+fi
```

Comment:
I'm clearly a novice with bash scripts am guessing there may be a DRYer more elegant way to set the `tag` in the body vs `branch`